spec/models/distribution_spec.rb

```diff
@@ -67,32 +67,28 @@
   end
 
   context "Methods >" do
-    before(:each) do
-      @distribution = create(:distribution)
-      @first = create(:item, name: "AAA", category: "Foo")
-      @last = create(:item, name: "ZZZ", category: "Bar")
-    end
+    let(:distribution) { create(:distribution) }
+    let(:item) { create(:item, name: "AAA", category: "Foo") }
+    let(:donation) { create(:donation) }
 
-    it "distributed_at" do
+    it "#distributed_at" do
       two_days_ago = 2.day.ago
       expect(create(:distribution, issued_at: two_days_ago).distributed_at).to eq(two_days_ago.strftime('%B %-d %Y'))
       expect(create(:distribution).distributed_at).to eq(Time.zone.now.strftime('%B %-d %Y'))
     end
 
-    it "copy_line_items" do
-      @donation = create(:donation)
-      @donation.line_items << create(:line_item, item: @first, quantity: 5)
-      @donation.line_items << create(:line_item, item: @first, quantity: 10)
-      expect(@distribution.copy_line_items(@donation.id).count).to eq 2
+    it "#copy_line_items" do
```

Comment:
Switched to using betterspecs.org convention on naming - can remove if undesirable.
Read more: http://www.betterspecs.org/#describe

Reply:
Nice, I like betterspecs. Thanks!

Reply:
TBH I think it should actually be 
`describe "#copy_line_items"` and then the `it` example line actually describe what the behavior is supposed to be. I realize that I'm probably the one that wrote it as `it "copy_line_items"` initially. I'll make the tweak.